src/commands/actions.js

```
@@ -137,6 +137,9 @@ module.exports.DeployActionCommand = class {
                 actionInst.command = options.cmd;
             }
             if (options.port) {
+                if (actionInst.type === 'job') {
+                    printError('--port not valid on job action types');
```

Thinking if we should throw an error or a warning.

---

FWIW, `printError()` by default does `process.exit(1)` after displaying the message, whereas `printWarning()` does **not** use process.exit.